cypress/integration/login.js

```diff
@@ -20,5 +20,6 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('[data-test="timesheets-current-day"]').contains(15)
```

Comment:
1. is this locator unique? [data-test="timesheets-current-day"]
2. Your test will fail tomorrow, so maybe there's a better way to check the date? e.g. a hint if you'll get stuck https://www.w3schools.com/jsref/jsref_getdate.asp